src/components/MainContainer/MainContainer.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+//import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import styled from 'styled-components';
+
+const MainContainerStyled = styled.section`
+  width: 100%;
+  min-height: 100vh;
+  background: #fff;
+  margin: 0;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const MainContainer = (props) => {
+  return (
+    <MainContainerStyled data-testid="main-container-component">
+      {props.children}
```

Comment:
This is just a thought, maybe the MainContainer.component could have in it an H1 and we could delete the Heading.component, unless the Heading.component will be reusable in other components this suggestion is not applicable.

Reply:
I have decided to keep it separately for the time being, since I am considering having a Login page, rather than a modal being displayed on top of the current view and this `Heading` could be useful there. Since I haven't fully discarded this option yet, I think it's sensible to keep it this way for now. What do you think?